docs/local-development.md

```diff
@@ -156,6 +156,8 @@ The app will start using that Geolite2 file for geolocation after restart.
 
 ### Testing on a mobile device or in a virtual machine
 
+[More detailed moble device instructions are here](mobile.md)
```

Comment:
Since the new document basically has all this information and more, does it make sense to just remove the section altogether, so that it doesn't fall out of sync or mislead someone to miss some information from the other document?

Reply:
I hesitated to do this only because this section mentioned virtual machines, which I didn't cover. Thoughts?

Reply:
Ah, good point. Not sure how common VM development would be these days since we no longer support Internet Explorer. Another angle could be to generalize the new article to "accessing the IdP on the network", though I think you have some useful platform-specific device guidance.

No strong preference, maybe one of:

- Remove references to "mobile device" in the section
- Go ahead and remove the section anyways, assuming VM development is uncommon enough